data/json/items/tools.json

```
@@ -90,7 +90,33 @@
     "color": "light_gray",
     "ammo": "battery",
     "max_charges": 1000,
-    "flags": [ "NO_UNLOAD", "RECHARGE" ]
+    "covers": [ "TORSO" ],
+    "encumbrance": 9,
+    "coverage": 15,
+    "material_thickness": 4,
+    "flags": [ "NO_UNLOAD", "RECHARGE", "BELTED", "UPS", "OVERSIZE" ]
+  },
+  {
+    "id": "UPS_Engineer_off",
+    "type": "TOOL_ARMOR",
+    "name": "engineer's UPS",
+    "name_plural": "engineers' UPS's",
```

```suggestion
    "name_plural": "engineer's UPS",
```
Plural name should be the same as singular in this case.